.swiftlint.yml

```diff
@@ -164,7 +164,10 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - focus-ios/
   # Ecosia: Exclude Swift Packages
   - SourcePackages
```

Comment:
I guess this can be removed in favor of the line below. Wonder why it did not complain before 🤔